Game/resources/Shaders/shaders.json

```diff
@@ -2,68 +2,68 @@
    "Shaders":[
       {
          "Name":"Skybox",
-         "Vertex":"Resources/Shaders/skybox.vs",
-         "Fragment":"Resources/Shaders/skybox.fs",
+         "Vertex":"/Resources/Shaders/skybox.vs",
+         "Fragment":"/Resources/Shaders/skybox.fs",
```

Comment:
The path now is different from Game?

Reply:
Is game, but when its mounte using PhysFS its transformed to /. So, Game is like the root path of everything